src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -42,10 +42,8 @@ function ProviderSelectionField({
     0,
     providersListLength,
   );
-  const providersListEmpty = currentlyShownProvidersList?.length === 0;
   const loadingProviders =
-    requestStatus === FETCH_STATUS.loading ||
-    requestStatus === FETCH_STATUS.notStarted;
+    !communityCareProviderList && requestStatus !== FETCH_STATUS.failed;
```

Comment:
I changed the way this loading state is defined because we would flash the 0 providers found alert in between fetching the current location and fetching providers.